bot/cogs/token_remover.py

```diff
@@ -86,6 +75,23 @@ def mod_log(self) -> ModLog:
                 channel_id=Channels.mod_alerts,
             )
 
+    def is_token_in_message(self, msg: Message) -> bool:
```

Comment:
This should probably be a classmethod, so you can call it from the codeblock cog without passing the class as the first argument, whilst still having access to the other methods.